webapp/courses/static/courses/terms.css

```diff
@@ -43,26 +43,53 @@ div.term-description:before {
     height: 0;
     position: absolute;
     width: 0;
-    top: -18px;
-    left: 0px;
-    border: 9px solid transparent;
-    border-bottom-color: #ccc;
+    top: -23px;
+    left: -1px;
+    border: 11px solid transparent;
+    border-bottom-color: #999;
 }
 
 div.term-description:after {
     content: ' ';
     height: 0;
     position: absolute;
     width: 0;
-    top: -16px;
-    left: 1px;
-    border: 8px solid transparent;
+    top: -19px;
+    left: 0px;
+    border: 10px solid transparent;
     border-bottom-color: #eee;
 }
 
+/* The arrow for left aligned description */
+div.term-description-left-aligned:before {
+    left: auto;
+    right: -1px;
+}
+
+div.term-description-left-aligned:after {
+    left: auto;
+    right: 0px;
```

Comment:
Using 0 (without the px) is customary, when the value is 0. (Also saves two bytes, but that's inconsequential...)